src/hugit/app/views.cljs

```diff
@@ -304,28 +304,45 @@
       :border {:type :line}
       :label (str " " @label " ")}
      [scrollable-list
-      {:top 0
-       :left 1
-       :right 2
-       :align :left
+      {:top         0
+       :left        1
+       :right       2
+       :align       :left
        :window-size (- rows 6)
-       :items @branches
+       :items       @branches
        :item-props-f
-       (fn [branch]
-         (if (= @current-branch branch)
-           {:style {:fg :green}}
-           {:style {:fg :white}}))
-       :selected @selected
+                    (fn [branch]
+                        (if (= @current-branch branch)
+                          {:style {:fg :green}}
+                          {:style {:fg :white}}))
+       :selected    @selected
        :custom-key-handlers
-       {["enter"] {:f (fn [idx]
-                        (let [branch (nth @branches idx)]
-                          (toast> "Checking out " branch)
-                          (evt> [:checkout-branch branch])
-                          (evt> [:router/goto :status])))
-                   :label "Checkout"
-                   :type "Action"}}
-       :on-back
-       #(evt> [:router/go-back])}]]))
+                    {["enter"] {:f (fn [idx]
```

Comment:
fix indentation

Reply:
this indentation is incorrect. `:custom-key-handlers` and `{["enter"] {:f (fn [idx] ...` should be aligned.